tests/lct_test.py

```diff
@@ -20,12 +20,97 @@
 ]
 
 
-def test_lct_signal():
+class _MultiCases:
+    def __init__(self):
+        self._values = self._vals()
+        self._signal = self._sigs()
+        for d in pkunit.case_dirs():
+            f = getattr(self, f"_{d.basename}")
```

Comment:
Dispatch methods should always have `_case` in front of them as I had written.

Reply:
my bad